vendor/bat-native-confirmations/include/bat/confirmations/confirmation_type.h

```diff
@@ -6,19 +6,34 @@
 #ifndef BAT_CONFIRMATIONS_CONFIRMATION_TYPE_H_
 #define BAT_CONFIRMATIONS_CONFIRMATION_TYPE_H_
 
+#include <string>
+
 namespace confirmations {
 
-static char kConfirmationTypeClick[] = "click";
-static char kConfirmationTypeDismiss[] = "dismiss";
-static char kConfirmationTypeView[] = "view";
-static char kConfirmationTypeLanded[] = "landed";
-
-enum class ConfirmationType {
-  UNKNOWN,
-  CLICK,
-  DISMISS,
-  VIEW,
-  LANDED
+class ConfirmationType {
+ public:
+  enum Value : int {
+    UNKNOWN,
+    CLICK,
+    DISMISS,
+    VIEW,
+    LANDED
+  };
+
+  ConfirmationType() = default;
+  constexpr ConfirmationType(const Value value) : value_(value) {}
```

Comment:
DITTO.

Reply:
👍 added NOLINT and comment as above, thanks